extension/src/background/helpers/account.ts

```diff
@@ -64,16 +64,16 @@ export const getIsFuturenet = async () => {
 };
 
 export const getIsMemoValidationEnabled = async () =>
-  (await localStore.getItem(IS_VALIDATING_MEMO_ID)) || true;
+  (await localStore.getItem(IS_VALIDATING_MEMO_ID)) ?? true;
```

Comment:
now that these values are bools and no longer string "true" or "false", when the value is set to `false`, it was defaulting to the backup case of `true`